roles/template_service_broker/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
 # placeholder file?
```

Comment:
Does `template_service_broker_remove` need a default value here as well?

Reply:
Isn't it on line 3?

Reply:
It's on line 3.

Reply:
Oops.